package/src/index.ts

```diff
@@ -70,6 +114,260 @@ export class CandleClient {
   }): Promise<string> {
     return this.candle.executeTool(tool);
   }
+
+  public async getAssetAccounts(
+    query: AssetAccountQuery = {
+      linkedAccountIDs: undefined,
+      assetKind: undefined,
+    }
```

Comment:
#nth this would be a linter error with our normal linter settings–omitting a field is different from setting its value to undefined, and in your definition of `AssetAccountQuery`, you used the former syntax (`assetKind?: string`)
```suggestion
    query: AssetAccountQuery = {}
```